label_studio/io_storages/urls.py

```diff
@@ -62,6 +63,13 @@
     path('export/redis/<int:pk>', RedisExportStorageDetailAPI.as_view(), name='export-storage-redis-detail'),
     path('export/redis/validate', RedisExportStorageValidateAPI.as_view(), name='export-storage-redis-validate'),
     path('export/redis/form', RedisExportStorageFormLayoutAPI.as_view(), name='export-storage-redis-form'),
+
+    # Local files
+    path('localfiles', LocalFilesImportStorageListAPI.as_view(), name='storage-localfiles-list'),
+    path('localfiles/<int:pk>', LocalFilesImportStorageDetailAPI.as_view(), name='storage-localfiles-detail'),
```

Comment:
Maybe local-files?

Reply:
We can't do it, because we use dynamic url structure based on python module names and we'll get incompatibility between `local-files`(denied in module name) and `local_files`(denied in urls).